### PR TITLE
Add name to some 'it' blocks in ProjectController

### PR DESCRIPTION
**Pull request by mdeniz**

### CODE REVIEW

**Comment by mdeniz** [comment]
@bgeuken please review

File: src/api/spec/controllers/webui/project_controller_spec.rb [hunk 1]
```diff
@@ -272,7 +272,7 @@
   end
 
   describe 'GET #new_package_branch' do
-    it do
+    it 'branch the package' do
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/project_controller_spec.rb]: *branches

File: src/api/spec/controllers/webui/project_controller_spec.rb [hunk 2]
```diff
@@ -321,7 +321,7 @@
 
   describe 'GET #add_repository_from_default_list' do
     context 'with some distributions' do
-      it do
+      it 'show repositories from default list' do
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/project_controller_spec.rb]: shows

**Comment by bgeuken** [comment]
LGTM. Feel free to merge after updating the two it blocks.

**Pull request merged**